app/controllers/api/notifications_controller.rb

```diff
@@ -9,7 +9,13 @@ def index
 
       serialized = SerializePaginatedCollection.(
         notifications,
-        SerializeUserNotifications
+        SerializeUserNotifications,
+        meta: {
```

Comment:
I added the ff:
- `links: all` in order to have a link to view all notifications
- `unread_count` in order to have the count of all unread notifications for the notification icon.

Is this okay?

Reply:
Yep - this is fine :)